### DISCUSSION TITLE
Thanks ElMassimo!

### DISCUSSION THREAD

**Discussion by chmich** [ElMassimo/vite_ruby#260] [Show and tell]
ElMassimo, great thanks to the work you`ve done!

Spent weeks for understanding all that new options around rails-7 frontend and bringing them all together to work.
My new rails-setup is ready and it wouldnt be possbile without your gems and npm packages.
I wrote tutorials for all.
they include: vite, inertia, svelte, Turbo, Bootstrap(sass), Foundation-Sites(sass), Stimulus, Svelte.
I will publish them as soon i can.

BR,
Christian

**Comment by ElMassimo** [comment]
Thanks Christian!

Let me know once you publish them, if you'd like we could link to them from the Vite Ruby docs.

**Reply by chmich** [reply]
look: https://dev.to/chmich/setup-vite-svelte-inertia-stimulus-bootstrap-foundation-on-rails-7-overview-1bk1

**Reply by chmich** [reply]
And yes, thanks, i am happy if you link it, i spent a lot of time for it and i hope it helps to inspire more developers.
